compiler+runtime/src/cpp/jank/runtime/obj/character.cpp

```diff
@@ -0,0 +1,38 @@
+#include <jank/runtime/obj/character.hpp>
+#include <jank/util/character.hpp>
+
+namespace jank::runtime
+{
+  obj::character::static_object(native_persistent_string_view const &d)
+    : data{ native_persistent_string(d) }
```

Comment:
`data` is a `native_persistent_string`, so there's no need to build another `native_persistent_string` here. What you're doing is:

1. Building a `native_persistent_string` from `d`
2. Building `data` from that temporary `native_persistent_string`

It's less efficient and it's more code than we need: `data{ d }`